mlflow/pyfunc/backend.py

```diff
@@ -231,6 +235,7 @@ def _execute_in_conda_env(
                         If False, return the server process `Popen` instance immediately.
     :param stdout: Redirect server stdout
     :param stderr: Redirect server stderr
+    :param stream_output_during_creating_env: stream output during running "conda env create" command.
```

Comment:
Can we clarify what happens when `stream_output` is `True` and what happens when it's `False`? It's not very obvious from reading the code right now. Specifically, are we relying on the fact that passing `stream_output=False` to `mlflow.utils.process.exec_cmd()` captures and returns the stdout / stderr streams? If so, the naming of the variable is also pretty confusing, since both modes are streaming output; the only difference is the destination streams: sys.stderr / sys.stdout vs subprocess.PIPEs.